src/index.html

```diff
@@ -22,6 +22,382 @@
+          alt="Moyo-Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops-computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <section
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__block">
+          <img
+            src="./images/imac.jpeg"
+            alt="product card image"
+            class="product-card__image"
+          />
+        </div>
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__product-code">Product code: 195434</div>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="product-card__reviews-count">Reviews: 5</div>
+        </div>
+
+        <div class="product-card__price-block">
+          <p class="product-card__price-text">Price:</p>
+
+          <p class="product-card__price">$2,199</p>
+        </div>
+
+        <button
+          class="product-card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
```

Comment:
The product cards are repeating multiple times with the same content. If these are placeholder cards for future dynamic content, it's okay. Otherwise, consider populating them with unique content or remove the duplicates if they are not needed.